docs/source/cpp/compute.rst

```diff
@@ -1056,7 +1056,8 @@ Temporal component extraction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 These functions extract datetime components (year, month, day, etc) from timestamp type.
-Note: this is currently not supported for timestamps with timezone information.
+Note: supports timestamps with timezone information and will return localized timestamp
+components.
```

Comment:
Rewrite this as (for example): "If the input timestamps have a non-empty timezone, localized timestamp components will be returned".

Reply:
Done.